src/ui/pages/Identifiers/Identifiers.tsx

```diff
@@ -279,9 +262,9 @@ const Identifiers = () => {
                 <IdentifierCardList
                   cardsData={pendingIdentifiers}
                   cardTypes={CardType.IDENTIFIERS}
-                  onCardClick={async (identifier) =>
-                    handlePendingClick(identifier as IdentifierShortDetails)
-                  }
+                  onCardClick={() => {
```

Comment:
Hey @Sotatek-HocNguyena could you please make this `onCardClick` prop as optional so we won't need to `return` it and we can just remove it from here.

Reply:
Ok, I will make the change

Reply:
updated! @sdisalvo-crd